Makefile

```diff
@@ -1,5 +1,74 @@
-all:
-	cargo build --release
+ifeq ($(OS),Windows_NT)
```

Comment:
does this actually work?

Reply:
I haven't tested on Windows, but seems to work on Mac / Linux.

Reply:
Simplified.